### PR TITLE
[TASK] Remove `DeclarationBlock::setSelector`

### PR DESCRIPTION
**Pull request by ziegenberg**
Removing one function after another.

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -21,6 +21,7 @@ This project adheres to [Semantic Versioning](https://semver.org/).
 ### Removed
 
 - Drop support for PHP < 7.2 (#420)
+- Final removal of `DeclarationBlock::setSelector()` (#560)
```
Comment (oliverklee) [inline CHANGELOG.md:24]: Our changelog is in reverse chronological order, i.e., please add new entries at the top of the list, not at the bottom. (Ditto for the other PRs that come with a changelog entry.)

And a note on commit messages/PR titles/changelog entries: Those should use the imperative voice. In this case:

> Remove `DeclarationBlock::setSelector()`

I found this article on git commit messages very helpful (and fun to read!): https://cbea.ms/git-commit/

We can also drop the "final" there as we already have the distinction between "deprecate …" and "remove …".

Ditto for the other PRs.

**Review by oliverklee — CHANGES_REQUESTED** [review]

File: CHANGELOG.md (hunk 1 above)
Comment (ziegenberg) [reply CHANGELOG.md:24]: I will revise the changelog messages.

> And a note on commit messages/PR titles/changelog entries: Those should use the imperative voice. In this case:
> 
> > Remove `DeclarationBlock::setSelector()`
> 
> I found this article on git commit messages very helpful (and fun to read!): https://cbea.ms/git-commit/

Can this be put in a CONTRIBUTING.md or similar file? It seems like every project has its own very helpful blog article someone found, (which is also fun to read), and they all contradict each other in one way or another. I wasn't able to find any notes on how to contribute to this project so I tried to infer the unwritten rules from what I found via `git log`.

I will update the commit messages as requested.

Comment (oliverklee) [reply CHANGELOG.md:24]: > Can this be put in a CONTRIBUTING.md or similar file?

Excellent suggestion!

I've added it to https://github.com/MyIntervals/PHP-CSS-Parser/issues/489#issuecomment-2176154113.

Comment (ziegenberg) [reply CHANGELOG.md]: I fixed the changelog message.

**Comment by ziegenberg** [comment]
I'm not going to fix the commit message, as that would require a force-push to have any effect, and that would contradict the desired fix-up commit. But I will remember the commit message format for future PRs.

**Review by oliverklee — APPROVED** [review]
Thanks! :pray:

**Pull request merged**